Cleanup & rework README.md

hboyd2003 opened the pull request:
Fully reworked README.md to look better and work better on different sized displayed.
@ImprovedTube Stuff like Planned Features and other things are not fully filled out as I am not exactly sure what to put there.

ImprovedTube commented:
@hboyd2003 sure, can still accept the PR and redo potential edits.  Might first clear obsolete/redundant commit history or readme.md       (  \&nbsp; _could_ be useful if you had to try without %, px or em  😀 )

ImprovedTube commented:
@hboyd2003 sure, can still accept the PR and redo potential edits.  Might first clear all obsolete/redundant commit history of readme.md       (  `&nbsp;` 😆 _could_ be useful like `em` _if_ you had to try without `%` or `px`    )

ImprovedTube commented:
@hboyd2003 sure, can still accept the PR and redo potential edits.  Might first clear all obsolete/redundant commit history of readme.md

hboyd2003 commented:
@ImprovedTube  Why was it reverted?

hboyd2003 commented:
@ImprovedTube I'm confused at what you did. Did you merge the pull request then just copy and paste the old one? I had reorganized the README.md with tabs and fixed many dead links and fixed reworked many of the paragraphs to make more sense... Most of which is not in the new README.md. Could you go back to my version an I'll add the changes you made. Then you add what else you want?

hboyd2003 commented:
@ImprovedTube  
> &nbsp; could be useful if you had to try without %, px or em
Ah! That's very useful I didn't know that's what you where using in place of that's. I spent an hour trying to get the tables to be smaller. I'll do some testing and see if I can put them back in.

ImprovedTube commented:
@hboyd2003  seen all 🤩 thank you.   not seriously attached to the spaces 😆 
just wanted to remove the ~800 obsolete historic commits of readme.md before accepting the PR

hboyd2003 commented:
@ImprovedTube  So, should I make a new pull request?

ImprovedTube commented:
yes, can/will accept every PR now, to document them properly 
(can also go back to your version from now for now)
(and/or can  make you a maintainer later)

ImprovedTube commented:
yes @hboyd2003 can/will accept every PR now, to document them properly
(can also go back to your version from now for now)
(and/or can make you a maintainer later)
Looking forward to see you as a top 2 contributor, now that the number of commits is fixed.

##### Readme.md:
 
- replaced your  \<ul\>\<li\> with markdown lists. (this works with a line break above)

- commented the current state with 😃
\<!--    * * * * * * * * * *   --\>
\<!--   For quick editing,  --\> 
\<!--  Text appears at the start of lines  --\>
\<!--   while html appears at the end  --\>
\<!--    * * * * * * * * * *   --\>

- The Translator (Credits) table is mainly to feature the Github names,    will try also shortening the language names to the ISO codes (and requires less width without the `<code>` around the check.)  🤔
 
- Projects list:
   - the regex replace repo has to be created from the Webstore Version
   - the striked through ones are not currently online, which doesn't make them less relevant for who reads that far 
     ( @ImprovedTube  and/or @victor-savinov will fill gaps)

hboyd2003 commented:
@ImprovedTube
> Looking forward to see you as a top 2 contributor, now that the number of commits is fixed.

I hope so!

>Readme.md:

Most of my cleanup still has been removed.... (and my commit that cleaned everything up and rework was never merged) Go to my version and turn off word-wrap and you'll should see all the cleanup I did.

Just delete my commit and I'll implement the changes you made and make a new pr. Then you can edit it to make it how you want.
